internal/webhooks/machinedeployment_webhook_test.go

```diff
@@ -532,14 +546,14 @@ func TestMachineDeploymentClusterNameImmutable(t *testing.T) {
 // make sure defaulting webhooks also pass validation tests on create, update and delete.
 // Note: The difference to util/defaulting.DefaultValidateTest is that this function takes an additional
 // CustomDefaulter as the defaulting is not implemented on the object directly.
-func defaultValidateTestCustomDefaulter(object admission.Validator, customDefaulter admission.CustomDefaulter) func(*testing.T) {
+func defaultValidateTestCustomDefaulter(object runtime.Object, webhook Webhook) func(*testing.T) {
```

Comment:
Please check if we can now delete this func and use CustomDefaultValidateTest instead

(I think we should add the warning assertions to util.CustomDefaultValidateTest)


I think we only needed this because we had the combination of Validator and CustomDefaulter, but now we have CustomValidator and CustomDefaulter